package.json

```diff
@@ -1,14 +1,15 @@
 {
-  "name": "vivi-type-proxy",
-  "version": "1.0.0",
+  "name": "type-proxy",
+  "version": "1.0.1",
   "description": "Validate data with unknown structure",
-  "author": "Andrew Pritchard <andrew@vivi.io>",
-  "private": true,
+  "author": "Vivi International",
+  "license": "MIT",
```

Comment:
This license should be mentioned in the README, and a separate LICENSE file included in the repo.